src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js

```diff
@@ -0,0 +1,362 @@
+    const clientLocalGroups = clientInfo.section.localGroups;
+    const localGroupPopup = mainPage.section.localGroupPopup;
+    // Open SUT and check that page is loaded
+    frontPage.navigate();
+    browser.waitForElementVisible('//*[@id="app"]');
+
+    // Enter valid credentials
+    frontPage
+      .clearUsername()
+      .clearPassword()
+      .enterUsername(browser.globals.login_usr)
+      .enterPassword(browser.globals.login_pwd)
+      .signin();
+
+    // Open TestGov Internal Servers and local group details view
+    mainPage.openClientsTab();
+    browser.waitForElementVisible(clientsTab);
+    clientsTab.openTestService();
+    browser.waitForElementVisible(clientInfo);
+    clientInfo.openLocalGroupsTab();
+    browser.waitForElementVisible(clientLocalGroups);
+    clientLocalGroups.openAbbDetails();
+    browser.waitForElementVisible(localGroupPopup);
+
+    // Add new member to local group
+    localGroupPopup.openAddMembers();
+    localGroupPopup.searchMembers();
+    localGroupPopup.selectNewTestComMember();
+    localGroupPopup.cancelAddMembersDialog();
+
+    //localGroupPopup.addSelectedMembers();
+    browser.waitForElementNotVisible('//span[contains(@class, "headline") and contains(text(), "Add Members")]');
+    browser.waitForElementVisible(localGroupPopup);
+    browser.assert.not.elementPresent('//*[contains(text(),"TestCom")]')
+
+    localGroupPopup.openAddMembers();
+    localGroupPopup.searchMembers();
+    localGroupPopup.selectNewTestComMember();
+    localGroupPopup.addSelectedMembers();
+
+    browser.waitForElementNotVisible('//span[contains(@class, "headline") and contains(text(), "Add Members")]');
+    browser.waitForElementVisible(localGroupPopup);
+    browser.assert.elementPresent('//*[contains(text(),"TestCom")]')
+
+    browser.end();
+
+  },
+  'Security server delete local group members': browser => {
+    const frontPage = browser.page.ssFrontPage();
+    const mainPage = browser.page.ssMainPage();
+    const clientsTab = mainPage.section.clientsTab;
+    const clientInfo = mainPage.section.clientInfo;
+    const clientLocalGroups = clientInfo.section.localGroups;
+    const localGroupPopup = mainPage.section.localGroupPopup;
+    // Open SUT and check that page is loaded
+    frontPage.navigate();
+    browser.waitForElementVisible('//*[@id="app"]');
+
+    // Enter valid credentials
+    frontPage
+      .clearUsername()
+      .clearPassword()
+      .enterUsername(browser.globals.login_usr)
+      .enterPassword(browser.globals.login_pwd)
+      .signin();
+
+    // Open TestGov Internal Servers and local group details view
+    mainPage.openClientsTab();
+    browser.waitForElementVisible(clientsTab);
+    clientsTab.openTestService();
+    browser.waitForElementVisible(clientInfo);
+    clientInfo.openLocalGroupsTab();
+    browser.waitForElementVisible(clientLocalGroups);
+    clientLocalGroups.openBacDetails();
+    browser.waitForElementVisible(localGroupPopup);
```

Comment:
Maybe even this part, also?

Reply:
The way how tests navigate to the actual target area of the SUT is going to be changed when the page model is refactored, so I'm not changing it at this time.